src/api/api_datastores.rs

```diff
@@ -0,0 +1,307 @@
+// This file is part of MinSQL
```

Comment:
Is it necessary to prefix the filename with `api_`? Let's not use unnecessary conventions.

Reply:
since I have other 2 PRs that rely on this file, let me fix this on a separate PR after #108 is merged